schemas/com.snowplowanalytics.snowplow/mobile_context/jsonschema/1-0-3

```diff
@@ -0,0 +1,126 @@
+			"type": "string",
+			"description": "The current version of the operating system"
+		},
+		"deviceManufacturer": {
+			"type": "string",
+			"description": "The manufacturer of the product/hardware"
+		},
+		"deviceModel": {
+			"type": "string",
+			"description": "The end-user-visible name for the end product"
+		},
+		"carrier": {
+			"type": ["string", "null"],
+			"description": "The carrier of the SIM inserted in the device"
+		},
+		"networkType": {
+			"type": ["string", "null"],
+			"enum": ["mobile", "wifi", "offline", null],
+			"description": "Type of network the device is connected to"
+		},
+		"networkTechnology": {
+			"type": ["string", "null"],
+			"description": "Radio access technology that the device is using"
+		},
+		"openIdfa": {
+			"type": ["string", "null"],
+			"description": "Deprecated tracking identifier for iOS"
+		},
+		"appleIdfa": {
+			"type": ["string", "null"],
+			"description": "Advertising identifier on iOS"
+		},
+		"appleIdfv": {
+			"type": ["string", "null"],
+			"description": "UUID identifier for vendors on iOS"
+		},
+		"androidIdfa": {
+			"type": ["string", "null"],
+			"description": "Advertising identifier on Android"
+		},
+		"physicalMemory": {
+			"type": ["integer", "null"],
+			"minimum": 0,
+			"maximum": 9223372036854775807,
+			"description": "Total physical system memory in bytes"
+		},
+		"systemAvailableMemory": {
+			"type": ["integer", "null"],
+			"minimum": 0,
+			"maximum": 9223372036854775807,
+			"description": "Available memory on the system in bytes (Android only)"
+		},
+		"appAvailableMemory": {
+			"type": ["integer", "null"],
+			"minimum": 0,
+			"maximum": 9223372036854775807,
+			"description": "Amount of memory in bytes available to the current app (iOS only)"
+		},
+		"batteryLevel": {
+			"type": ["integer", "null"],
+			"minimum": 0,
+			"maximum": 100,
+			"description": "Remaining battery level as an integer percentage of total battery capacity"
+		},
+		"batteryState": {
+			"type": ["string", "null"],
+			"enum": ["unplugged", "charging", "full", null],
+			"maxLength": 255,
+			"description": "Battery state for the device"
+		},
+		"lowPowerMode": {
+			"type": ["boolean", "null"],
+			"description": "A Boolean indicating whether Low Power Mode is enabled (iOS only)"
+		},
+		"availableStorage": {
+			"type": ["integer", "null"],
+			"minimum": 0,
+			"maximum": 9223372036854775807,
+			"description": "Bytes of storage remaining"
+		},
+		"totalStorage": {
+			"type": ["integer", "null"],
+			"minimum": 0,
+			"maximum": 9223372036854775807,
+			"description": "Total size of storage in bytes"
+		},
+		"isPortrait": {
+			"type": ["boolean", "null"],
+			"description": "A Boolean indicating whether the device orientation is portrait (either upright or upside down)"
+		},
+		"resolution": {
+			"type": ["string", "null"],
+			"maxLength": 20,
+			"description": "Screen resolution in pixels. Arrives in the form of WIDTHxHEIGHT (e.g., 1200x900)"
+		},
```

Comment:
Given this is device information, I think I'd imagine it to stay static, but I'm curious to know if you'd expect it to change as device orientation changes or stay the same? Might be worth adding the behvaiour to the description.

Reply:
Good point! It doesn't change with the orientation, I added a note in description.